TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
+    user = 'X';
+    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+    messages = "Game - START!";
+    complete = false;
+
+    reset() {
+        this.moves = 0;
+        this.user = 'X';
+        this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+        this.messages = "Game - START!";
+        this.complete = false;
+    }
+
+    getMoves() {
```

Comment:
I think the use of getters here may be superfluous, since the members are public anyways. I say just access `game.moves` directly -- getters like this are a Java-ism more than anything.

If you want to be pedantic, JavaScript does have the idea of getters and setters, as well as private members, which can allow you to achieve a similar effect. Something like (I don't know the syntax too well so there may be some mistakes here):
```javascript
class A {
  #private_mem_variable = 42;

  get meaning_of_life() {
    return this.#private_mem_variable;
  }
}

let a = new A();
console.log(a.meaning_of_life);
// Note: not `a.meaning_of_life()`, no parens: getters and setters allow you to imitate the behavior of ordinary data members
```